packages/fxa-content-server/app/tests/spec/views/mixins/sync-optional-mixin.js

```diff
@@ -13,7 +13,7 @@ import sinon from 'sinon';
 
 class View extends BaseView {
   template() {
-    return '<a href="#" id="do-not-sync-device">Do not Sync</a>';
+    return '<a href="#" id="do-not-sync-device" class="link-blue">Do not Sync</a>';
```

Comment:
Probably unrelated to this exact PR, but does this need to be localized?

Reply:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/13018240/181116599-b6f8c2e0-409f-4865-823e-8940e0e6cc02.png">

Reply:
@pdehaan Actually I reverted this change. This is in a test file so we don't need the class or l10n.